Gemfile

```diff
@@ -11,7 +11,7 @@ if RUBY_VERSION >= "2.3.0"
     gem 'guard-rspec'
   end
 else
-  gem "rails", ">= 3.0", "< 5.0"
+  gem "rails", ">= 3.0", "<= 5.1"
```

Comment:
This should perhaps be Ruby-version dependent (i.e. `< 5.0` for Ruby < 2.2.2 or 2.3, else `< 5.2`) so Travis CI builds and developers on older Ruby versions install a compatible version and builds on newer versions get the latest available.